### PR TITLE
portal.json in Spanish, optimized translation

### PR DESCRIPTION
**Pull request by fochoa8**
Syntaxis checking, proof reading, translation, and explanations without too much IT technology terminology, but regular expressions, not only limited to Latin America, but Spanish speakers worldwide.

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [ ] There's a clear use-case for this code change, explained below
- [x] Commit message has a short title & references relevant issues
- [ ] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

### CODE REVIEW

File: ghost/i18n/locales/es/portal.json [hunk 1]
```diff
@@ -6,171 +6,153 @@
...
-    "Billing info": "Información de pago",
+    "Back to Log in": "Atrás para Iniciar Sesión",
+    "Billing info": "Información de Pago",
     "Black Friday": "Black Friday",
-    "Cancel anytime.": "Cancelación en cualquier momento.",
+    "Cancel anytime.": "Cancela en cualquier momento.",
     "Cancel subscription": "Cancelar suscripción",
     "Cancellation reason": "Razón de cancelación",
     "Change": "Cambiar",
-    "Change plan": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:28]: this is a needed string. Please don't delete.

File: ghost/i18n/locales/es/portal.json [hunk 2]
```diff
@@ -6,171 +6,153 @@
...
     "Check spam & promotions folders": "Comprueba la carpeta de spam y promociones",
-    "Check with your mail provider": "Comprueba con tu proveedor de correo",
+    "Check with your mail provider": "Comprueba con tu proveedor de correo electrónico",
     "Choose": "Elige",
     "Choose a different plan": "Elige un plan diferente",
-    "Choose a plan": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:33]: needed string, please don't delete.

File: ghost/i18n/locales/es/portal.json [hunk 3]
```diff
@@ -6,171 +6,153 @@
     "{{amount}} off for first {{period}}.": "{{amount}} de descuento por los primeros {{period}}.",
     "{{amount}} off forever.": "{{amount}} de descuento para siempre.",
     "{{discount}}% discount": "{{discount}}% de descuento",
-    "{{memberEmail}} will no longer receive {{newsletterName}} newsletter.": "{{memberEmail}} ya no recibirá boletines de {{newsletterName}}.",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:9]: error discussed in your previous PR.  No accent on the plural.

File: ghost/i18n/locales/es/portal.json [hunk 4]
```diff
@@ -6,171 +6,153 @@
...
+    "{{memberEmail}} will no longer receive {{newsletterName}} newsletter.": "{{memberEmail}} ya no recibirá boletínes de {{newsletterName}}.",
     "{{memberEmail}} will no longer receive emails when someone replies to your comments.": "{{memberEmail}} ya no recibirá correos electrónicos cuando alguien responde a tus comentarios.",
     "{{memberEmail}} will no longer receive this newsletter.": "{{memberEmail}} ya no recibirá este boletín.",
-    "{{trialDays}} days free": "{{trialDays}} días gratis",
-    "+1 (123) 456-7890": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:13]: Please don't remove strings.

File: ghost/i18n/locales/es/portal.json [hunk 5]
```diff
@@ -6,171 +6,153 @@
...
     "Don't have an account?": "¿No tienes una cuenta?",
     "Edit": "Editar",
     "Email": "Correo electrónico",
     "Email newsletter": "Boletín informativo por correo electrónico",
-    "Email preferences": "Preferencias",
+    "Email preferences": "Preferencias de correo electrónico",
     "Emails": "Correos electrónicos",
     "Emails disabled": "Correos electrónicos desactivados",
     "Ends {{offerEndDate}}": "Termina el {{offerEndDate}}",
-    "Enter your email address": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:59]: these strings are in use. please do not delete.

File: ghost/i18n/locales/es/portal.json [hunk 6]
```diff
@@ -6,171 +6,153 @@
...
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "Añade este correo electrónico {{senderEmail}} a tu lista de conactos confiados. Esto le indica a tu provedor de correo electrónico, que puedes recibir correos de esta dirección de correo sin problemas.",
     "Less like this": "Menos como esto",
     "Make sure emails aren't accidentally ending up in the Spam or Promotions folders of your inbox. If they are, click on \"Mark as not spam\" and/or \"Move to inbox\".": "Asegúrate de que los correos electrónicos no terminen accidentalmente en las carpetas de correo no deseado o promociones de su bandeja de entrada. Si lo son, haz clic en \"Marcar como no spam\" y/o \"Mover a la bandeja de entrada\".",
     "Manage": "Administrar",
     "Maybe later": "Tal vez más tarde",
     "Memberships unavailable, contact the owner for access.": "Membresía no disponible, contacta al propietario para obtener acceso.",
-    "month": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:88]: please do not delete - this string is needed to show the monthly price. (i.e. $5/month in English.)

File: ghost/i18n/locales/es/portal.json [hunk 7]
```diff
@@ -6,171 +6,153 @@
...
     "Not receiving emails?": "¿No recibes correos electrónicos?",
     "Now check your email!": "¡Ahora revisa tu correo electrónico!",
-    "Once resubscribed, if you still don't see emails in your inbox, check your spam folder. Some inbox providers keep a record of previous spam complaints and will continue to flag emails. If this happens, mark the latest newsletter as 'Not spam' to move it back to your primary inbox.": "Una vez que te hayas vuelto a suscribir, si aún no ves los correos electrónicos en tu bandeja de entrada, verifica tu carpeta de spam. Algunos proveedores de bandejas de entrada mantienen un registro de quejas de spam anteriores y continuarán marcando los correos electrónicos. Si esto sucede, marca el último boletín como \"No es spam\" para moverlo nuevamente a tu bandeja de entrada principal.",
-    "Permanent failure (bounce)": "Fallo permanente (bounce)",
-    "Phone number": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:98]: Please don't delete strings.

File: ghost/i18n/locales/es/portal.json [hunk 8]
```diff
@@ -6,171 +6,153 @@
...
+    "Permanent failure (bounce)": "Fallo permanente (rebote de correo al remitente)",
     "Plan": "Plan",
     "Plan checkout was cancelled.": "El pago de plan fue cancelado.",
-    "Plan upgrade was cancelled.": "La actualización de plan fue cancelada.",
-    "Please contact {{supportAddress}} to adjust your complimentary subscription.": "Por favor, ponte en contacto con {{supportAddress}} para ajustar tu suscripción gratuita.",
-    "Please enter {{fieldName}}": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:103]: deleted string?

File: ghost/i18n/locales/es/portal.json [hunk 9]
```diff
@@ -6,171 +6,153 @@
...
-    "Send an email to {{senderEmail}} and say hello. This can also help signal to your mail provider that emails to and from this address should be trusted.": "¡Envia un correo electrónico a {{senderEmail}} y saluda! Esto también puede ayudar a indicarle a su proveedor de correo que se debe confiar en los correos electrónicos hacia y desde esta dirección.",
+    "Send an email to {{senderEmail}} and say hello. This can also help signal to your mail provider that emails to and from this address should be trusted.": "¡Envia un correo electrónico a {{senderEmail}} y saluda! Esto también puede ayudar a su proveedor de correo que debe confiar en los correos electrónicos hacia y desde esta dirección de correo.",
     "Sending login link...": "Enviando enlace de inicio de sesión...",
     "Sending...": "Enviando...",
     "Show all": "Mostrar todo",
     "Sign in": "Iniciar sesión",
     "Sign out": "Cerrar sesión",
     "Sign up": "Registrarse",
     "Signup error: Invalid link": "Error de registro: Enlace inválido",
-    "Something went wrong, please try again later.": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:120]: Please don't delete strings.

File: ghost/i18n/locales/es/portal.json [hunk 10]
```diff
@@ -6,171 +6,153 @@
...
+    "Submit feedback": "Enviar comentarios de retroalimentación",
     "Subscribe": "Suscribir",
     "Subscribed": "Suscrito",
     "Success": "Éxito",
     "Success! Check your email for magic link to sign-in.": "¡Éxito! Revisa tu correo electrónico para ver el enlace mágico para iniciar sesión.",
     "Success! Your account is fully activated, you now have access to all content.": "¡Éxito! Tu cuenta está completamente activada, ahora tienes acceso a todo el contenido.",
     "Success! Your email is updated.": "¡Éxito! Tu correo electrónico está actualizado.",
     "Successfully unsubscribed": "Te has dado de baja correctamente",
-    "Thank you for subscribing. Before you start reading, below are a few other sites you may enjoy.": "Gracias por suscribirte. Antes de leer, aquí abajo hay otros sitios que te pueden gustar.",
-    "Thank you for your support": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:135]: please don't delete strings.

File: ghost/i18n/locales/es/portal.json [hunk 11]
```diff
@@ -6,171 +6,153 @@
...
-    "Thank you for your support!": "",
-    "Thanks for the feedback!": "¡Gracias por tus comentarios!",
+    "Thank you for subscribing. Before you start reading, below are a few other sites you may enjoy.": "Gracias por suscribirte. Antes de que empieces a leer, aquí abajo hay otros sitios que te pudieran gustar.",
+    "Thanks for the feedback!": "¡Gracias por tus comentarios de retroalimentación!",
     "That didn't go to plan": "Eso no salió según lo planeado",
-    "The email address we have for you is {{memberEmail}} — if that's not correct, you can update it in your <button>account settings area</button>.": "La dirección de correo electrónico que tenemos para ti es {{memberEmail}} — si no es correcta, puedes actualizarla en tu <button>área de configuración de la cuenta</button>.",
-    "There was a problem submitting your feedback. Please try again a little later.": "Hubo un problema al enviar tus comentarios. Vuelve a intentarlo un poco más tarde.",
-    "There was an error processing your payment. Please try again.": "",
-    "This site is invite-only, contact the owner for access.": "Este sitio es solo por invitación, contacta al propietario para obtener acceso.",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:142]: please check for deleted strings.

File: ghost/i18n/locales/es/portal.json [hunk 12]
```diff
@@ -6,171 +6,153 @@
...
+    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "Cuando una bandeja de entrada no acepta un correo electrónico, se le llama rebote de correo electrónico. En muchos casos, esto puede ser temporal, dado que el correo puede ser añadido a lista blanca. Sin embargo, en algunos casos, un correo electrónico rebota por una falla permanente, dado que dicha dirección no es válida o no existe.",
     "Why has my email been disabled?": "¿Por qué mi correo electrónico ha sido desabilitado?",
-    "year": "",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:163]: this string gives price per year. Please don't delete.

**Review by cathysarisky — CHANGES_REQUESTED** [review]

File: ghost/i18n/locales/es/portal.json (hunk 3 above)
Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json:9]: No, I meant boletines no accents, boletín does have accent.

Comment (cathysarisky) [reply ghost/i18n/locales/es/portal.json:9]: Agreed. But you removed the correct bolotines and changed it to boletínes with this PR.

Comment (cathysarisky) [reply ghost/i18n/locales/es/portal.json:9]: Agreed that boletines should not have an accent.  However, your PR removes boletines on line 9 and changes it to boletínes.

**Comment by fochoa8** [comment]
Updated it, check the lines, 178 lines, all properly translated, thanks for warning me and not passing the review carelessly. And fact checking too, shows Your interest and level of engagement in an open source project.

File: ghost/i18n/locales/es/portal.json (hunk 6 above)
Comment (cathysarisky) [reply ghost/i18n/locales/es/portal.json:88]: resolved.

File: ghost/i18n/locales/es/portal.json (hunk 7 above)
Comment (cathysarisky) [reply ghost/i18n/locales/es/portal.json:98]: resolved :)

File: ghost/i18n/locales/es/portal.json [hunk 13]
```diff
@@ -9,168 +9,168 @@
...
     "Memberships unavailable, contact the owner for access.": "Membresía no disponible, contacta al propietario para obtener acceso.",
-    "month": "",
+    "month": "mes",
     "Monthly": "Mensual",
     "More like this": "Más como esto",
     "Name": "Nombre",
     "Need more help? Contact support": "¿Necesitas más ayuda? Contacta a soporte técnico.",
-    "Newsletters can be disabled on your account for two reasons: A previous email was marked as spam, or attempting to send an email resulted in a permanent failure (bounce).": "Los boletines se pueden deshabilitar en tu cuenta por dos razones: Un correo electrónico anterior se marcó como correo no deseado o el intento de enviar un correo electrónico resultó en una falla permanente (bounce).",
+    "Newsletters can be disabled on your account for two reasons: A previous email was marked as spam, or attempting to send an email resulted in a permanent failure (bounce).": "Los boletines se pueden deshabilitar en tu cuenta por dos razones: Un correo electrónico anterior se marcó como correo no deseado o el intento de enviar un correo electrónico resulta en una falla permanente (rebota el correo al remitente).",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:93]: I think that the past tense (resultó ) is probably more correct here?  We're saying that the email bounced in the past, not that the email is bouncing now (because we have given up trying).

File: ghost/i18n/locales/es/portal.json [hunk 14]
```diff
@@ -9,168 +9,168 @@
...
-    "Something went wrong, please try again later.": "",
+    "Something went wrong, please try again later.": "Algo no funcionó, por favor intenta mas tarde.",
     "Sorry, that didn’t work.": "Lo siento, eso no funcionó.",
     "Spam complaints": "Quejas de spam",
-    "Start {{amount}}-day free trial": "Comienza el período de prueba gratuito de {{amount}} días",
+    "Start {{amount}}-day free trial": "Comienza el período de prueba gratuito de {{amount}} días gratis",
     "Starting {{startDate}}": "A partir de {{startDate}}",
     "Starting today": "A partir de hoy",
-    "Submit feedback": "Enviar comentarios",
+    "Submit feedback": "Enviar comentarios de retroalimentación",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:126]: This is text for a button, so unless you feel that the shorter version is wrong, perhaps we can revert this change?

File: ghost/i18n/locales/es/portal.json [hunk 15]
```diff
@@ -9,168 +9,168 @@
...
-    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "Cuando una bandeja de entrada no acepta un correo electrónico, comúnmente se le llama rebote. En muchos casos, esto puede ser temporal. Sin embargo, en algunos casos, un correo electrónico rebotado puede devolverse como una falla permanente cuando una dirección de correo electrónico no es válida o no existe.",
+    "When an inbox fails to accept an email it is commonly called a bounce. In many cases, this can be temporary. However, in some cases, a bounced email can be returned as a permanent failure when an email address is invalid or non-existent.": "Cuando una bandeja de entrada no acepta un correo electrónico, se le llama rebote de correo electrónico. En muchos casos, esto puede ser temporal, dado que el correo puede ser añadido a lista blanca. Sin embargo, en algunos casos, un correo electrónico rebota por una falla permanente, dado que dicha dirección no es válida o no existe.",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:161]: Whitelisting is a separate concept that is not in the original text.  Perhaps remove that phrase?

**Comment by cathysarisky** [comment]
> Updated it, check the lines, 178 lines, all properly translated, thanks for warning me and not passing the review carelessly. And fact checking too, shows Your interest and level of engagement in an open source project.

Thank you for all your work on this!  This is much improved.  I'm sorry to have left just a few more questions and comments. :)

File: ghost/i18n/locales/es/portal.json (hunk 3 above)
Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json:9]: oh did not notice, sorry to be slow to respond, have been real busy.

File: ghost/i18n/locales/es/portal.json (hunk 14 above)
Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json:126]: Oh got it, sure.

**Comment by fochoa8** [comment]
[portal.json](https://github.com/user-attachments/files/17193625/portal.json)

Fixed.

**Comment by fochoa8** [comment]
Did what You asked me to do referring to portal.json, want assistance for all the rest of translation, of all 4 files?

**Comment by fochoa8** [comment]
Here comments.json file properly translated

[comments.json](https://github.com/user-attachments/files/17193806/comments.json)

**Comment by fochoa8** [comment]
Will do the rest later, I am busy at work, thanks for doing the revisions.

File: ghost/i18n/locales/es/portal.json (hunk 14 above)
Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json]: Did it already.

File: ghost/i18n/locales/es/portal.json (hunk 15 above)
Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json]: Got it, deleted it.

**Comment by fochoa8** [comment]
> > Updated it, check the lines, 178 lines, all properly translated, thanks for warning me and not passing the review carelessly. And fact checking too, shows Your interest and level of engagement in an open source project.
> 
> Thank you for all your work on this! This is much improved. I'm sorry to have left just a few more questions and comments. :)

No problem, I like contributing to projects, knew about Ghost when I studied in the MIT. One final exam was related to Ghost blog, to these days the best blog system I have ever seen.

**Comment by cathysarisky** [comment]
I appreciate that it is easier to link an updated file than to actually get the updates into the PR, but your changes will not get merged unless you get them into the PR.  In this case, the file you provided has an error, making it especially important that it goes through all the testing.  Thanks!

**Comment by fochoa8** [comment]
> I appreciate that it is easier to link an updated file than to actually get the updates into the PR, but your changes will not get merged unless you get them into the PR. In this case, the file you provided has an error, making it especially important that it goes through all the testing. Thanks!

Got it.

**Comment by fochoa8** [comment]
Better I send all translations in one batch, max 10pm UTC -6 Central time

**Comment by fochoa8** [comment]
Here is the new PR https://github.com/TryGhost/Ghost/pull/21167

**Comment by fochoa8** [comment]
> I appreciate that it is easier to link an updated file than to actually get the updates into the PR, but your changes will not get merged unless you get them into the PR. In this case, the file you provided has an error, making it especially important that it goes through all the testing. Thanks!

Here is the full translation to Spanish https://github.com/TryGhost/Ghost/pull/21167

**Comment by sagzy** [comment]
Closing this PR, as duplicate of https://github.com/TryGhost/Ghost/pull/21167
